client/src/components/GlobalResultsGrid.tsx

```diff
@@ -49,6 +50,14 @@ export const GlobalResultsGrid = withStyles(mainStyles, { withTheme: true })(
                         label="people solved at least one test set"
                     />
                 </Grid>
+                <Grid item xs={12} sm={12} md={12} style={{ textAlign: "center" }}>
+                    <Typography variant="h5" gutterBottom>
+                        Participation by country
+                    </Typography>
+                    <WorldMap
+                        groupsByParticipants={getNumberOfParticipantsByCountry(results)}
```

Comment:
Could the prop have a different name to make this component easier to reuse? I think something as simple as `data` would work better than `groupsByParticipants` here since the map could in theory visualise not only participant data, but arbitrary numbers.